# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/koules/mkdirhier.patch
# Copyright (C) 2004 - 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

We do not have this legacy mkdirhier anymore ...

  - Rene Rebe <rene@exactcode.de>

Also install the custom targets into $(DESTDIR), and do not strip with the host strip.

--- koules1.4.orig/Imakefile	1998-03-04 19:59:19.000000000 +0100
+++ koules1.4/Imakefile	2026-10-18 02:39:35.785605281 +0200
@@ -98,23 +98,23 @@ install:: $(PROGRAMS) $(SOUNDSERVER)
 	@echo "Koules directory is " $(KOULESDIR)
 	@:
 	@echo "Instalin man page to " $(MANDIR)
-	@mkdirhier $(MANDIR)
-	$(INSTALL) -c $(INSTBINFLAGS) xkoules.6 $(MANDIR);
+	@mkdir -p $(DESTDIR)$(MANDIR)
+	$(INSTALL) -c $(INSTBINFLAGS) xkoules.6 $(DESTDIR)$(MANDIR);
 	@:
 	@echo "Creating directory " $(KOULESDIR)
-	@if [ ! -d $(KOULESDIR) ]; then mkdirhier $(KOULESDIR); fi
+	@if [ ! -d $(DESTDIR)$(KOULESDIR) ]; then mkdir -p $(DESTDIR)$(KOULESDIR); fi
 	@:
 	@if [ "$(INSTALLSOUND)" = "True" ]; 				\
 	 then								\
 		echo "Creating directory " $(SOUNDDIR); 		\
-		if [ ! -d $(SOUNDDIR) ]; then mkdirhier $(SOUNDDIR); fi;    \
+		if [ ! -d $(DESTDIR)$(SOUNDDIR) ]; then mkdir -p $(DESTDIR)$(SOUNDDIR); fi;    \
 		echo "Copying sound server to directory " $(SOUNDDIR); \
-		$(INSTALL) -c -s $(INSTBINFLAGS) $(SOUNDSERVER) $(SOUNDDIR); \
+		$(INSTALL) -c $(INSTBINFLAGS) $(SOUNDSERVER) $(DESTDIR)$(SOUNDDIR); \
 		echo "Copying sound files into directory " $(SOUNDDIR); \
 		cd ./sounds; set -x;					\
 		for file in *.raw; do                                   \
 		    auconvert -raw linu8 1 -rate 8000 -volume 40 $$file -file snd `basename $$file .raw`.au ; \
-		    $(INSTALL) -c $(INSTDATFLAGS) `basename $$file .raw`.au $(SOUNDDIR); \
+		    $(INSTALL) -c $(INSTDATFLAGS) `basename $$file .raw`.au $(DESTDIR)$(SOUNDDIR); \
 		done;                                                   \
 	 fi
 	@echo
@@ -126,23 +126,23 @@ install:: $(PROGRAMS) $(SOUNDSERVER)
 	@echo "Koules directory is " $(KOULESDIR)
 	@:
 	@echo "Instalin man page to " $(MANDIR)
-	@mkdirhier $(MANDIR)
-	$(INSTALL) -c $(INSTBINFLAGS) xkoules.6 $(MANDIR);
+	@mkdir -p $(DESTDIR)$(MANDIR)
+	$(INSTALL) -c $(INSTBINFLAGS) xkoules.6 $(DESTDIR)$(MANDIR);
 	@:
 	@echo "Creating directory " $(KOULESDIR)
-	@if [ ! -d $(KOULESDIR) ]; then mkdirhier $(KOULESDIR); fi
+	@if [ ! -d $(DESTDIR)$(KOULESDIR) ]; then mkdir -p $(DESTDIR)$(KOULESDIR); fi
 	@:
 	@if [ "$(INSTALLSOUND)" = "True" ]; 				\
 	 then								\
 		echo "Creating directory " $(SOUNDDIR); 		\
-		if [ ! -d $(SOUNDDIR) ]; then mkdirhier $(SOUNDDIR); fi;    \
+		if [ ! -d $(DESTDIR)$(SOUNDDIR) ]; then mkdir -p $(DESTDIR)$(SOUNDDIR); fi;    \
 		echo "Copying sound server to directory " $(SOUNDDIR); \
-		$(INSTALL) -c -s $(INSTBINFLAGS) $(SOUNDSERVER) $(SOUNDDIR); \
+		$(INSTALL) -c $(INSTBINFLAGS) $(SOUNDSERVER) $(DESTDIR)$(SOUNDDIR); \
 		echo "Copying sound files into directory " $(SOUNDDIR); \
 		cd ./sounds; set -x;					\
 		for file in *.raw; do                                   \
 		    sh ../myauconvert $$file `basename $$file .raw`.au ; \
-		    $(INSTALL) -c $(INSTDATFLAGS) `basename $$file .raw`.au $(SOUNDDIR); \
+		    $(INSTALL) -c $(INSTDATFLAGS) `basename $$file .raw`.au $(DESTDIR)$(SOUNDDIR); \
 		done;                                                   \
 	 fi
 	@echo
@@ -156,22 +156,22 @@ install:: $(PROGRAMS) $(SOUNDSERVER)
 	@echo "Koules directory is " $(KOULESDIR)
 	@:
 	@echo "Instalin man page to " $(MANDIR)
-	@mkdirhier $(MANDIR)
-	$(INSTALL) -c $(INSTBINFLAGS) xkoules.6 $(MANDIR);
+	@mkdir -p $(DESTDIR)$(MANDIR)
+	$(INSTALL) -c $(INSTBINFLAGS) xkoules.6 $(DESTDIR)$(MANDIR);
 	@:
 	@echo "Creating directory " $(KOULESDIR)
-	@if [ ! -d $(KOULESDIR) ]; then mkdirhier $(KOULESDIR); fi
+	@if [ ! -d $(DESTDIR)$(KOULESDIR) ]; then mkdir -p $(DESTDIR)$(KOULESDIR); fi
 	@:
 	@if [ "$(INSTALLSOUND)" = "True" ]; 				\
 	 then								\
 		echo "Creating directory " $(SOUNDDIR); 		\
-		if [ ! -d $(SOUNDDIR) ]; then mkdirhier $(SOUNDDIR); fi;    \
+		if [ ! -d $(DESTDIR)$(SOUNDDIR) ]; then mkdir -p $(DESTDIR)$(SOUNDDIR); fi;    \
 		echo "Copying sound server to directory " $(SOUNDDIR); \
-		$(INSTALL) -c -s $(INSTBINFLAGS) $(SOUNDSERVER) $(SOUNDDIR); \
+		$(INSTALL) -c $(INSTBINFLAGS) $(SOUNDSERVER) $(DESTDIR)$(SOUNDDIR); \
 		echo "Copying sound files into directory " $(SOUNDDIR); \
 		cd ./sounds; set -x;					\
 		for file in *.raw; do					\
-		    $(INSTALL) -c $(INSTDATFLAGS) $$file $(SOUNDDIR);	\
+		    $(INSTALL) -c $(INSTDATFLAGS) $$file $(DESTDIR)$(SOUNDDIR);	\
 		done;   						\
 	 fi
 	@echo
